frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -1302,6 +1305,8 @@ test("begins_typeahead", (override) => {
     assert_typeahead_equals("~~~ f", lang_list);
     assert_typeahead_equals("test\n~~~ p", lang_list);
     assert_typeahead_equals("test\n~~~  p", lang_list);
+    assert_typeahead_equals("```krok", lang_list);
+    assert_typeahead_equals("~~~kroki b", lang_list);
```

Comment:
I'd like to see a test that more obviously shows the kroki full syntax being suggested.